.gitignore

```diff
@@ -8,4 +9,5 @@ cross
 *.swp
 .idea
 *.iml
+*.test
```

Comment:
It appears to be per package rather than for the workspace as a whole. I need to find if there's some VSCode config option that lets me write them all to some other location (like the vim option that let's you put all the `~` files in a specific place) because the files are really annoying.